### PR TITLE
Netsim imports colors from shared color scss

### PR DESCRIPTION
**Pull request by islemaster**
As a proof-of-concept for using the shared color.scss file from the apps pipeline.

### CODE REVIEW

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -208,7 +208,8 @@ config.lodash = {
 config.sass = {
   all: {
     options: {
-      outputStyle: (MINIFY ? 'compressed' : 'nested')
+      outputStyle: (MINIFY ? 'compressed' : 'nested'),
+      includePaths: ['../shared/css/']
```
Comment (islemaster) [inline apps/Gruntfile.js]: Here's the new include path.  @wjordan is there a new version of this I need to modify as well?

File: shared/css/color.scss [hunk 2]
```diff
@@ -1,9 +1,11 @@
 // colors from https://www.dropbox.com/s/ycjiwt8vxlmv2ih/Code.org_stylesheet_v2_09.05.13.pdf
 
+$black: #000000;
 $charcoal: #5b6770;
 $light_gray: #949ca2;
 $lighter_gray: #c6cacd;
 $lightest_gray: #e7e8ea;
+$white: #ffffff;
```
Comment (islemaster) [inline shared/css/color.scss]: Figured we could use a pure black and pure white as well.  Apparently (says hound) good form is to always use variables for colors, even ones that have browser definitions.

**Comment by islemaster** [comment]
Of course, this changes the netsim colors a bit.  Now using official colors for packet header columns:

screenshot from 2015-04-09 17 41 58

**Comment by joshlory** [comment]
*2015 in history: April 9 – This pull request is opened.*

File: apps/Gruntfile.js (hunk 1 above)
Comment (wjordan) [inline apps/Gruntfile.js]: The new version is currently in the [frontend_pipeline](https://github.com/code-dot-org/code-dot-org/tree/frontend_pipeline) branch at [lib/frontend/sass.js](https://github.com/code-dot-org/code-dot-org/blob/frontend_pipeline/lib/frontend/sass.js), I've added a [commit](https://github.com/code-dot-org/code-dot-org/commit/0da9a7bfd9497d5ffa0d710bef1111a652515998) that matches this change.

Comment (islemaster) [inline apps/Gruntfile.js]: :+1: Thanks!

File: shared/css/color.scss (hunk 2 above)
Comment (laurelfan) [inline shared/css/color.scss]: could use $black: black; and $white: white; I guess

**Comment by wjordan** [comment]
:+1: shared code :heart_eyes:

File: shared/css/color.scss (hunk 2 above)
Comment (islemaster) [inline shared/css/color.scss]: True; maybe we should do this only for the [17 CSS2 colors](http://stackoverflow.com/a/8561913)?

I could go either way.

**Pull request merged**